test/test.xunit.assert/Asserts/FailAssertsTests.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+using Xunit.Sdk;
+
+namespace test.xunit.assert.Asserts
```

Comment:
Looking at the other tests, this isn't required

Reply:
Fair enough. Namespace removed.